### ISSUE TITLE
Order:page

### ISSUE THREAD

**Issue by QnF6eq** [funmaker/Hybooru#28]
Being able to order by page would be a nice addition - it'd be awesome to also have a _next_ and _previous_ button on the post page.

**Comment by funmaker** [comment]
Does Hydrus keep track of what pages belong to single collection/post?
I can add a functionality to order posts by `page:*` tags but then you'd probably have pages from various posts mixed together. Ie. first you'd see all `page:1` posts, then `page:2` posts, etc, so I doubt this would be any useful.
Is there any specific Hydrus feature that you have in mind that you'd want to see in Hybooru?

**Comment by damianb** [comment]
I suspect what they mean is a subsort.  Order by import time, secondary import by page if present, grouping by...*something*. Maybe related images?

**Comment by funmaker** [comment]
Well, the import time is going to differ between pages, so even if you do secondary sort by page, the final order will depend mostly on how hydrus imported the files.
I see that Hydrus has an option to sort by a sequence of tags like:
![image](https://user-images.githubusercontent.com/3338680/231538135-059aae75-caa2-4fef-b080-3a585c5f5e22.png)

I personally do not have use for such feature. But it wouldn't be hard to implement, I can give it a go, but first I'd like to know the exact usecase of it.

**Comment by QnF6eq** [comment]
Yeah you both are right - this is meant for a search like "series:example volume:1 order:page" since order by ID or import time is likely not the real page order.

This feature would excel if this would allow for ordering by any namespace - as to mimmic hydrus' behavior shown in your screenshot eg. "series:example order:volume order:page".

**Comment by funmaker** [comment]
Implemented in [v1.15.0](https://github.com/funmaker/Hybooru/releases/tag/v1.15.0)
